03_letters/davidbroughsmyth/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letterFreq(s string) map[rune]int {
+	f := map[rune]int{}
+	for _, r := range s {
+		f[r]++
+	}
+	return f
+}
+
+// Concurrent implementation of letters for speed
+// Returns the letter frequency for string
+func letters(s string) map[rune]int {
+	stringSlice := strings.Fields(s)
+	combineFreq := map[rune]int{}
```

Comment:
I'd suggest moving this closer to where it is used (just above the loop that reads the results). This looks too much like C-style - declaring vars at the top of the function, and is not idiomatic go - keeping the "range" of a variable short so you can stick to shorter names is the usual practice.

Reply:
I've done this change.